examples/todo-list/flutter-todo-list/lib/constant.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
+
+boxSmall() => const SizedBox(height: 20);
+boxLarge() => const SizedBox(height: 40);
+
+Widget buttonCustom(String text, Function() onPressed, bool filled) {
```

Comment:
It might be easier to read when we are calling it if we made these named parameters like this:

```suggestion
Widget buttonCustom({required String text, required void  Function() onPressed, required bool filled,}) {
```

Reply:
DONE ✅